test.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+from electr import Electr
+
+class TestHamster(unittest.TestCase):
```

Comment:
I'm not sure how we ended up in hamsters, and why do they fix electric wires in Vilshanka, but let's count it for a new species of hamsters :)